pages/[resourceType]/[id]/evaluate.tsx

```diff
@@ -35,37 +65,120 @@ const EvaluateMeasurePage = () => {
+                    label={<Text size="lg">Select a reportType</Text>}
+                    style={{ marginBottom: "25px" }}
+                  >
+                    <Radio value="Subject" label="Subject" />
+                    <Radio value="Population" label="Population" />
+                  </RadioGroup>
+
+                  {/* only displays autocomplete component if radio value is Patient */}
+                  {radioValue === "Subject" ? (
+                    <SelectComponent
+                      resourceType="Patient"
+                      setValue={setPatientValue}
+                      value={patientValue}
+                      required={true}
+                    />
+                  ) : (
+                    <SelectComponent
+                      resourceType="Patient"
+                      setValue={setPatientValue}
+                      value={patientValue}
+                      disabled={true}
+                      placeholder="Patient selection disabled when 'Population' is selected"
+                    />
+                  )}
+                </Grid.Col>
+
+                <Grid.Col span={3} style={{ margin: 5 }}>
+                  <SelectComponent
+                    resourceType="Practitioner"
+                    setValue={setPractitionerValue}
+                    value={practitionerValue}
+                  />
+                </Grid.Col>
+              </Grid.Col>
+              <Grid.Col span={3} style={{ margin: 5 }}>
+                <h3
+                  style={{
+                    color: textGray,
+                    marginTop: "20px",
+                    marginBottom: "2px",
+                    textAlign: "center",
+                  }}
+                >
+                  Request Preview:{" "}
+                </h3>
+                <div
+                  style={{
+                    textAlign: "center",
+                    overflowWrap: "break-word",
+                    padding: "10px",
+                    backgroundColor: "#F1F3F5",
+                    border: "1px solid",
+                    borderColor: "#4a4f4f",
+                    borderRadius: "20px",
+                    marginLeft: "30px",
+                    marginRight: "30px",
+                  }}
+                >
+                  <Text
+                    size="md"
+                    style={{ color: textGray, textAlign: "left" }}
+                  >{`${createRequestPreview()}`}</Text>{" "}
+                </div>
+              </Grid.Col>
+              <Grid.Col span={3} style={{ minHeight: 100, margin: 5 }}>
+                <div
+                  style={{
+                    textAlign: "center",
+                  }}
+                >
+                  <Button
+                    color="cyan"
+                    radius="lg"
+                    size="md"
+                    style={{
+                      textAlign: "center",
+                    }}
+                  >
+                    {" "}
```

Comment:
```suggestion
```
I also don't see this doing anything but I could be wrong! If it does help with spacing, I would try seeing what you can do by adding padding in div style or Button. If you tried that, this is probably fine, just looks a but cleaner :)